oximeter/db/build.rs

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright 2024 Oxide Computer Company
+
+use anyhow::Context as _;
+use nom::IResult;
+
+/// Build script for generating native type representations from the
+/// ground-truth SQL definitions.
+fn main() -> anyhow::Result<()> {
+    const INIT_FILE: &str =
+        concat!(env!("CARGO_MANIFEST_DIR"), "/schema/single-node/db-init.sql");
```

Comment:
Do we not have to worry about the multi-node schemas because we're just extracting the types here and aren't worried about table creation? I think a comment saying why we only need the single-node would be helpful.

Reply:
That's right, yeah. I just want the enum definitions for the columns here, which are the same in both single-node and replicated. I'll add a comment, good idea, thanks.

Reply:
Done in 997682132